iterators.tex

```diff
@@ -276,7 +276,7 @@
   template <class I>
   concept bool Readable@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ Semiregular<I>@\newtxt{()}@ &&
-      requires (I i) {
+      requires (@\newtxt{const }@I i) {
```

Comment:
The examples should use only built-ins. I also wonder if the committee will balk at our defining "semantically equivalent" to allow for observably different side-effects. But similar allowance is made for copy elision under the as-if rule. Hrm, note to self: look into the as-if rule and see if it can be used here.